ipatests/prci_definitions/prci_checker.py

```diff
@@ -0,0 +1,198 @@
+#! /usr/bin/python3
+
+import os
+import yaml
+import glob
+import click
+
+HINT = click.style("HINT: ", fg="bright_magenta")
+WARNING = click.style("WARNING: ", fg="yellow")
+ERROR = click.style("ERROR: ", fg="red")
```

Comment:
Can't say I'm a huge fan of colorized output. This is likely to appear as escape sequences in CI output on Azure.

Reply:
As this tool was intented to be just a manual tool for devs, I just thougth of it as nice feature for better UX. But of course, if we think about integrating it into some automated pipeline, I could delete that to avoid any trouble.

Also from [Click docs](https://click.palletsprojects.com/en/7.x/utils/#ansi-colors):

> Click’s [echo()](https://click.palletsprojects.com/en/7.x/api/#click.echo) function will automatically strip ANSI color codes if the stream is not connected to a terminal.